recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -1,2 +0,0 @@
-setup_conda_rc .\ .\recipe .\.ci_support\%CONFIG%.yaml
```

Comment:
we also need this unitl this https://github.com/conda-forge/conda-smithy/pull/934 is fixed.

Reply:
IMO by using a dev package this package should also be a dev package not published in `main`.

Reply:
One may argue that the other dev release is stable enough and out it on main instead. I'd rather have that than this workaround.

Reply:
I'm of the opinion that we simply follow the package itself as its' uploaded to pypi - if they see fit to call their own package `1.1.2` whilst depending on a beta version of a library I think we should simply follow their lead and publish to main.

If we publish to a dev label whilst pip users can simply install without a `--pre` it will raise questions/concerns about what the difference is between the pip and conda packages.

The reason we need this is because we published [`apispec`](https://github.com/conda-forge/apispec-feedstock) `1.0.0b1` to the dev label rather than main. In general, I think this is good practice because if we publish `1.0.0b5` to main and there are breaking changes then IIUC conda will pick the latest (`1.0.0b5` > `0.39.0`) and some people will be unhappy.

That said, marshmallow have an odd habit of publishing beta releases for ~years so I think many people (like `responder`) treat them as non-*really*-beta releases. Given that, I'd personally be happy to publish `1.0.0b5` to the main channel and force `0.39.0` users to pin if required.

Reply:
Case-in-point: `marshmallow` itself publishes its beta releases to the main channel

Reply:
I perfectly agree with your approach @dhirschfeld. At the moment, there is only [this](https://github.com/conda-forge/flask-apispec-feedstock) feedstock depending on `apispec`, so no problem.

Unless @ocefpaf has a _very_ strong argument against this, I would proceed and trash the whole `dev` label stuff (at least until it is better sorted out at the conda-forge/core level).

Reply:
IMO PyPI is completely broken by allowing that and is pushing bad practices downstream. I'm not opposed of merging this b/c it is up to the maintainers. But I would never do that for a package I maintain!

PS: this kind of workaround will get you in trouble with out infrastructure and the bot as you already notice below. Good luck :stuck_out_tongue_winking_eye:

Reply:
@ocefpaf I still don't get the full picture of you proposed solution: even if we push the package to the `dev` label, there is still the problem that it won't pick up `apispec` from `dev`, so we _need_ this wacky workaround (any better idea on this?) unless we push everything on `main`. Or what am I missing?

Reply:
Honestly I don't like upsetting the infrastructure and the 🤖, so until this custom label stuff is fixed and consolidated, I'd go for `main` given the fact that both packages seem pretty stable.

In general @ocefpaf I agree with you that beta is beta, so it should not go on release channels. But the problem here is that conda-forge is not ready for this.

Reply:
> But the problem here is that conda-forge is not ready for this.

I agree. That is why I prefer to send "beta" to `main`, when it is "stable enough" but we cannot get upstream to release it, than working around it :smile:

Reply:
@dhirschfeld I'll work on apispec as soon as I have some time